ocaml/utils/language_extension.mli

```diff
@@ -50,10 +50,15 @@ val is_erasable : 'a t -> bool
 
 (** Print and parse language extensions; parsing is case-insensitive *)
 val to_string : 'a t -> string
+val to_command_line_string : 'a t -> 'a -> string
 val of_string : string -> Exist.t option
 
 val maturity_to_string : maturity -> string
 
+(** Get the command line string enabling the given extension, if it's
+    enabled; otherwise None *)
+val get_enabled_command_line_string : 'a t -> string option
```

Comment:
Maybe `get_enabling_command_line_string`?

Reply:
Er.... `get_the_command_line_string_that_enabled_this_extension`? The enabling happened in the past, hence my choice. So I prefer my name over your suggestion, but am quite happy for further suggestions.

Reply:
`get_command_line_string_if_enabled`?  Reading this a day later, I don't think my initial suggestion is meaningfully better anymore